Makefile

```diff
@@ -49,3 +49,12 @@ teardown-local:
 	# This target tears down the environment deployed by deploy-local.
 	-helm delete --purge clair
 	-helm delete --purge clair-pg
+
+.PHONY: lint-proto
+lint-proto:
+	clang-format -i api/v3/clairpb/clair.proto
+
+
+.PHONY: gen-drone-config
+gen-drone-config:
+	jsonnet -y .drone.jsonnet -o .drone.yml
```

Comment:
I think we should make this `drone jsonnet --stream` since that's what their documentation says to use.